EIPS/eip-draft_tx_expir.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: <to be assigned>
+title: Transaction Expiration
+description: Proposes a Transaction Expiration Block for every transaction.
+author: Lucas Vinzon<lucasvinzon@gmail.com>
```

Comment:
```suggestion
author: Lucas Vinzon (@SnowPrimate)
```

At least one author needs to have a GitHub username. If you'd like to also include your email, you can add yourself twice, the second time with your email.